### PR TITLE
Add css quiz fixtures

### LINKED ISSUE #60: Add CSS Quiz fixtures
Todo
- [ ] Create the CSS quiz fixtures that have been posted on Instagram/Twitter
- [ ] Add PHPUnit tests

Follow the folder/file structure of the HTML Quiz.
[HTML Quiz fixtures:](https://github.com/howToCodeWell/code-quiz/tree/main/api/config/fixtures/quizzes/html-quiz)
[Twitter CSS questions](https://twitter.com/search?q=%40howtocodewell%20%23codequiz%20CSS&src=typed_query&f=top)

### PR DESCRIPTION
**Pull request by metinbaris**
- Fixes #60

### CODE REVIEW

File: api-client/src/mockData/cssQuiz/cssQuestions.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
+import {QuestionJsonldQuestionRead} from "../../generated/openapi"
+
+const CSSQuestions: QuestionJsonldQuestionRead[] = [
+    {
+        '@context': "/api/v1/contexts/Question",
+        '@id': "/api/v1/question/15",
```
Comment (metinbaris) [inline api-client/src/mockData/cssQuiz/cssQuestions.ts:6]: Hello @pfwd, got small confusion related ids on this points, should we keep them unique?

File: api-client/src/mockData/cssQuiz/cssQuiz.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {QuizJsonldRead} from "../../generated/openapi"
+
+const CSSQuiz: QuizJsonldRead =
+    {
+        '@context': "/api/v1/contexts/Quiz",
+        '@id': "/api/v1/quiz/4",
```
Comment (metinbaris) [inline api-client/src/mockData/cssQuiz/cssQuiz.ts:6]: Similar confusion here at id's

File: api-client/src/mockData/cssQuiz/cssQuiz.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        '@type': "Quiz",
+        title: "CSS Quiz",
+        slug: 'css-quiz',
+        questions: [
+            "/api/v1/question/15",
+            "/api/v1/question/16",
+            "/api/v1/question/17",
+            "/api/v1/question/18",
+            "/api/v1/question/19",
```
Comment (metinbaris) [inline api-client/src/mockData/cssQuiz/cssQuiz.ts:15]: These numbering also :)

File: api-client/src/mockData/cssQuiz/cssQuestions.ts (hunk 1 above)
Comment (pfwd) [reply api-client/src/mockData/cssQuiz/cssQuestions.ts:6]: Yes they should be unique. Ideally they should match those in the database but I'm not sure that will be possible once we start adding more.

Comment (pfwd) [reply api-client/src/mockData/cssQuiz/cssQuestions.ts:6]: Perhaps we need a script that will re-write the IDs.  I will need to give it some thought 🤔 
For now, please keep them as unique as you can.

Comment (metinbaris) [reply api-client/src/mockData/cssQuiz/cssQuestions.ts:6]: @pfwd ok thank you, I am just opening a request for this one we can handle it in an another issue maybe, it's hard with static values at this point

**Review by pfwd — APPROVED** [review]
Looks good @metinbaris thanks

**Pull request merged**